docs/src/selectors.md

```diff
@@ -118,7 +115,7 @@ A[X=Where(>=(1.5)), Y=Where(x -> x in (:a, :c))]
 
 == Not
 
-`Not(x)` get all indices _not_ selected by `x`, which can be another selector.
+The [`Not(x)`](@ref) selector gets all indices _not_ selected by `x`, which can be another selector.
```

Comment:
this one is failing in CI, not sure why....

Reply:
We don't own it so we can't use ref